Makefile

```diff
@@ -358,7 +359,7 @@ help:
 	@echo "  check      - Run Flake8 on sources"
 	@echo "  pylint     - Run PyLint on sources"
 	@echo "  installtest - Run install tests"
-	@echo "  safety      - Run Safety on minimum_constraints.txt"
+	@echo "  safety     - Run Safety for install and all"
```

Comment:
Suggest modifying to include full file names.  This helps to clarify that safety is run on the minimum constraints files.

Reply:
That would be too long for a half-line description. Do you have a proposal for something that fits into this half line?

Reply:
I tried several things an lets forget my comment.  This is really for us.